### PR TITLE
[conversations] Drop python 3.6

### PR DESCRIPTION
**Pull request by syso-jxx**
# Description

closes https://github.com/Azure/azure-sdk-for-python/issues/25166

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [X] **The pull request does not introduce [breaking changes]**
- [X] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by wonhyeongseo — APPROVED** [review]
All keyvault related content has been reverted.

File: sdk/cognitivelanguage/azure-ai-language-conversations/README.md [hunk 1]
```diff
@@ -12,13 +12,14 @@ Conversational Language Understanding - aka **CLU** for short - is a cloud-based
 ## _Disclaimer_
 
 _Azure SDK Python packages support for Python 2.7 ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```
Comment (kristapratico) [inline sdk/cognitivelanguage/azure-ai-language-conversations/README.md:15]: We put the special disclaimer for Python 2.7 to give customers a heads up that they may need to update their code, but we don't need to do the same for dropping Python 3.6. You can remove this comment.

**Review by kristapratico — COMMENTED** [review]
Thanks so much for your contribution! Once the review comment is resolved I can approve and merge your PR. :)

**Comment by wonhyeongseo** [comment]
Hello @syso-jxx, I have sent a PR that will resolve the review from the Azure team. May you please check it out? Thanks!

**Review by kristapratico — APPROVED** [review]
Awesome! Thanks for the help :)

**Pull request merged**